docs/source/1.0/spec/http-protocol-compliance-tests.rst

```diff
@@ -105,8 +105,9 @@ that support the following members:
         the query string (for example, "/foo/bar").
     * - host
       - ``string``
-      - The host / endpoint provided to the client, not including the path or
-        scheme (for example, "example.com").
+      - The host / endpoint provided to the client (for example, "example.com").
+        This will generally not include a path unless it is a non-modeled prefix,
```

Comment:
Can you flip this sentence around and say when it would a path, an example of what it looks like, and how it works with paths in operations? It's important to state that here since we aren't using the same URI resolution logic defined in [RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986#section-5).

For example:

> `host` MAY contain a path to indicate a base path from which each operation in the service is appended to. For example, given a `host` of `example.com/foo/bar` and an operation path of `/MyOperation`, the resolved host of the operation is `example.com` and the resolved path is `/foo/bar/MyOperation`.